software/messaging/src/main/java/brooklyn/entity/messaging/rabbit/RabbitSshDriver.java

```diff
@@ -24,6 +24,7 @@
 import java.util.List;
 import java.util.Map;
 
+import com.google.api.client.repackaged.com.google.common.base.Strings;
```

Comment:
I'm tempted to set Checkstyle to completely forbid `com.google.api.client.repackaged`.